src/main/java/org/cryptomator/integrations/common/IntegrationService.java

```diff
@@ -0,0 +1,20 @@
+package org.cryptomator.integrations.common;
+
+public interface IntegrationService {
```

Comment:
This seems like an abstraction for other services as well. Or why was this introduced? Do we need that? Why should revealing a path have a display name?

Reply:
The idea is not bad, but we should keep this in a separate PR on a later version.

Reply:
This interface is intended for documenation dedup and the general possibility to adjust the used service in GUIs.

Yes it does not directly belong to this PR. It is not needed.

The display name is needed when the user wants to select a special service implementation. "revealing a path" is just a feature, but each service implementation would implement it differently. And depending on the system config, the user might prefer one over another. (thinking of Linux).

But none the less, i reverted it in https://github.com/cryptomator/integrations-api/pull/16/commits/d3f9a24456ff71bfdf6c4af3e32ad7f076bf043f